spec/replace_browser_extension/01_main.rst

```diff
@@ -0,0 +1,542 @@
+========  ===========================  ================  ==========
+epic      title                        author            created
```

Comment:
@tschubotz do we put author to know whom to contact right from the doc (by the example of eip) or no?

Reply:
It doesn't hurt I think, let's add author and inital date, yes. I'll add it to my PRs as well.